tools/api-markdown-documenter/api-report/api-markdown-documenter.api.md

```diff
@@ -376,6 +376,17 @@ export interface HtmlRenderContext extends TextFormatting {
     prettyFormatting?: boolean;
 }
 
+declare namespace HtmlRenderer {
```

Comment:
Do we have to use namespaces? They're very weird from a JS perspective.

Reply:
If they must be used, please add a comment explaining why this usage is OK.

Reply:
I totally missed that this is the epi-extractor output, so this may be irrelevant.

Reply:
They aren't really namespaces, they're module exports. This is just how API-Extractor interprets them.

Reply:
I _think_ API-Extractor treats them this way because semantically module exports serve the same role, even if the implementation details are different.